manifests/controllerconfig.crd.yaml

```diff
@@ -1106,6 +1760,8 @@ spec:
                 type: string
             required:
             - additionalTrustBundle
+            - baseOSContainerImage
+            - baseOSExtensionsContainerImage
```

Comment:
`baseOSExtensionsContainerimage`  was optional because some things (like OKD) don't ship extensions, but this newly regenerated CRD was just generated off of our types.go, and we didn't tag it as optional there and we probably need to:
https://github.com/openshift/machine-config-operator/blob/ff47fa1185e6ba3e30e642f48300b6c250d0d66f/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L77-L80

I mean, I don't think it _technically_ matters since:
- everything that touches this structure does so through the go client 
- the field is not tagged as `omitempty` on the go side, (so our updates are going to include the field regardless, albeit maybe as a blank string "").  

but yes, to do this properly, types.go would need an `// +optional` for that field. 

(something something api migration :smile: )

Reply:
got it. fine with me as long as we are not breaking :)